src/openrct2/ride/ride_ratings.c

```diff
@@ -904,10 +904,10 @@ static uint32 ride_ratings_get_proximity_score()
 }
 
 /**
- * Seems to calculate how much of the track is sheltered in eighths.
+ * Calculates how much of the track is sheltered in eighths.
  *  rct2: 0x0065E72D
  */
-static sint32 sub_65E72D(rct_ride *ride)
+static sint32 get_num_of_sheltered_eighths(rct_ride *ride)
```

Comment:
See https://github.com/OpenRCT2/OpenRCT2/blob/develop/src/openrct2/ride/ride.h#L225
I've called this `undercover_portion` but perhaps a rename to `sheltered_eigths` would be better.